src/api/app/controllers/webui/labels_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Webui::LabelsController < Webui::WebuiController
+  before_action :set_project
+  before_action :set_labelable
+
+  def update
+    authorize @labelable, :update?
```

Comment:
The `:update?` is not really needed, Pundit will detect the controller action and call the appropriate policy method

Reply:
This is now `update_labels?`, which is not the same as the controller action